### DISCUSSION TITLE
Are Gemspec/RequiredRubyVersion issues with `start_column > last_column` intended?

### DISCUSSION THREAD

**Discussion by radomirbosak** [rubocop/rubocop#10241] [Q&A] [answered]
I believe the following behavior started with version [1.22.2](https://github.com/rubocop/rubocop/releases/tag/v1.22.2) .

When I create a spec file `foo.gemspec` like this
```ruby
Gem::Specification.new do |spec|
  spec.name = 'foo'
end
```
And run rubocop (e.g. latest 1.22.3)
```json
$ rubocop --format json foo.gemspec  | jq '.files[0].offenses[0]'
{
  "severity": "convention",
  "message": "Gemspec/RequiredRubyVersion: `required_ruby_version` should be specified.",
  "cop_name": "Gemspec/RequiredRubyVersion",
  "corrected": false,
  "correctable": false,
  "location": {
    "start_line": 1,
    "start_column": 1,
    "last_line": 1,
    "last_column": 0,
    "length": 0,
    "line": 1,
    "column": 1
  }
}
```
Then I get a situation where `start_column > last_column`, a contradiction on first sight. Is this intended?

Tools which further process rubocop output might be confused by this.

In version 1.21.0, both `start_column` and `last_column` were equal to `1`.

**Comment by koic** [comment, marked as answer]
This looks like a regression. I opened https://github.com/rubocop/rubocop/pull/10242 PR.
